db/controllers/news.controller.js

```diff
@@ -27,10 +27,29 @@ exports.getAllArticles = (request, response, next) => {
 }
 
 exports.getArticleCommentsByID = (request, response, next) => {
-    const id = request.params.article_id
-    selectCommentsByArticleID(id)
+    const article_id = request.params.article_id
+    selectCommentsByArticleID(article_id)
     .then((comments) => {
         response.status(200).send({ comments : [comments]})
     })
     .catch(next);
+}
+
+exports.postComment = (request, response, next) => {
+    const article_id  = request.params.article_id;
+    const { username, body } = request.body;
+    selectUserByUsername(username)
```

Comment:
you don't need to do this, as an author that doesn't exist will result in a psql error.